h2o-docs/src/product/data-science/upliftdrf.rst

```diff
@@ -230,7 +231,9 @@ You can set the AUUC type to be computed:
 
 In ``auuc`` the default AUUC is stored, however you can see also AUUC values for other AUUC types in ``auuc_table``.
 
-The resulting AUUC value is not normalized, so the result could be a positive number, but also a negative number. A higher number means better model.
+The resulting AUUC value is not normalized, so the result could be a positive number, but also a negative number. A higher number means better model. 
+
+To get normalized AUUC, you have to call ``auuc_normalized`` method. The normalized AUUC is calculated from uplift values which are normalized by uplift value from maximal treated number of observations. So if you have for example uplift values [10, 20, 30] the normalized uplift is [1/3, 2/3, 1]. It the maximal value is negative, the normalization factor is absolute value from this number. The result normalized AUUC can be again negative and positive and can be outside of (0, 1) interval. 
```

Comment:
**IF** the maximal value is negative, the normalization factor is **the** absolute value from this number. The normalized AUUC can be again negative and positive and outside of (0, 1) interval. 

Kudos gramarly :)